CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.2] 2022-05-02
+
+### Changed
+
+- Expose `bulk_update` function to the native `mongodb::Collection` via an Extension Trait `CollectionExt`
+
+## [0.8.1] 2022-04-30
```

Comment:
I think the only change for 0.8.1 was this PR: https://github.com/Devolutions/mongodm-rs/pull/17